theme/parts/icons.css

```diff
@@ -338,3 +338,7 @@ menuitem[type="radio"][disabled="true"] .menu-iconic-icon {
 		list-style-image: var(--webextension-menupanel-image-light, inherit) !important;
 	}
 }
+
+.bookmark-item[container] {
```

Comment:
You will also need to add this to the icons color statement from `icons.css#L6`.

Reply:
And done!